### DISCUSSION TITLE
Вычитание по истории удаляет больше чем нужно

### DISCUSSION THREAD

**Discussion by KisoidPoh095** [Chimildic/YaMuTools#28] [Помощь с ошибкой] [answered]
Приветствую!

У меня есть плейлист из 2155 композиций. Из них 53 загружены мною с локального компа, т.к. их нет и не предвидится в каталоге ЯМузыки. Я называю такие треки "приватными", т.к. они видны только мне.

Если запустить очистку плейлиста по истории (Фильтр треков -> Удалить недавно игравшие: 500), то расширение удаляет не только те мелодии, которые входят в последние 500 прослушиваний, но и **все** приватные треки заодно.
Несмотря на то, что я их не слушал последнее время (проверено).

**Comment by Chimildic** [comment, marked as answer]
В версии 0.8.0 загруженные mp3 удалятся не должны

**Reply by kisoid** [reply]
УПС! Боюсь, что теперь эта функциональность **сломалась**...
Теперь, если сделать удаление 500 недавних по истории (или любого другого кол-ва), то удаляются какие-то рандомные треки (не понятно, по какому принципу), а действительно недавно игравшие остаются...
Если после этого повторно выбрать удаление по истории и выбрать любое кол-во треков, то будет написано "Не обнаружено треков для удаления"

**Reply by kisoid** [reply]
UPD откатился на 0.7.6
там треки по истории удаляются (ну и соответственно, вернулась проблема с загруженными)
